src/components/HeaderThemeContainer/HeaderThemeContainer.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { ThemeToggle } from '../ThemeToggle/ThemeToggle';
+
+import './HeaderThemeContainer.scss';
+
+export function HeaderThemeContainer() {
+  const [theme, setTheme] = React.useState('dark');
+
+  function toggleTheme() {
+    setTheme(theme === 'dark' ? 'light' : 'dark');
+  }
+
+  React.useEffect(() => {
+    document.documentElement.setAttribute('data-theme', theme);
+  }, [theme]);
+
+  return (
+    <header className="header-theme-container">
+      <h1 className="h1-header">Where in The World?</h1>
```

Comment:
samo 'header' wystarczy - dopisek 'h1' nie dodaje wiele, a jedynie sprawia, że będzie trudniej w przypadku wprowadzania zmian, gdy np. stanie się to h2 a coś innego będzie h1